internal/models/versionanalyzers/helm_version_analyzer_test.go

```diff
@@ -116,3 +116,13 @@ func (s *HelmVersionAnalyzerTestSuite) TestVersionUpdateIsNeededWhenSomeChangesA
 	})
 	s.True(needsVersionUpdate)
 }
+
+func (s *HelmVersionAnalyzerTestSuite) TestNonHelmifiedRevision() {
+	s.NoError(os.MkdirAll("./test-dir", os.ModePerm))
+	helmRootDir := "./test-dir"
+	version := utils.GetResultOrPanic(s.helmVersionAnalyzer.ReadVersion(".", &models.ArgumentValues{
+		HelmArgumentKeyRootDir: &helmRootDir,
+	}))
+	s.Equal("", version)
+	s.NoError(os.RemoveAll("./test-dir"))
```

Comment:
Use teardown for such cleanings. Since this line may not run when a previous assertion fails.